src/python/update_container_definitions.py

```diff
@@ -45,13 +46,21 @@ def run(previous_task_definition, container_image_name_updates, container_env_va
                     raise ValueError('The container ' + container_name +
                                      ' is not defined in the existing task definition')
                 container_index = container_entry['index']
-                env_var_entry = container_entry['environment_map'].get(
-                    env_var_name)
+
+                # Exception is raised when trying to update env vars of a container with an empty env var list
```

Comment:
May I ask about the intention of this check? Is it to not allow adding new environment variables if the original container definition did not have any environment variables defined?

Reply:
@lokst this was added to specifically address a failing test... you can see it here: https://github.com/stringbeans/aws-ecs-orb/blob/remove-env-restriction/src/python/tests/test_update_container_definitions.py#L68

Reply:
@stringbeans Thanks for explaining the rationale! I feel that the restriction could be removed and that the test can be removed/modified as it would make sense to be consistent in allowing adding of env vars, whether or not there are currently any env vars defined. Would you mind if I pushed some changes to update_container_definitions.py and test_update_container_definitions.py, to your branch, to add on to your PR?

Reply:
@lokst go for it :)